qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -29,7 +29,6 @@
     storage_type_local = directory
     enable_ceph_mirror1 = yes
     enable_ceph_data1 = no
-    enable_ceph_image1 = no
```

Comment:
According to the comment in line 28, it's for local image. Why don't we add "enable_ceph_image1 = yes" under "ceph:"? like data1 in line 31 and line 33. If I was wrong, please correct me. Thanks.

Reply:
> hi @XueqiangWei , this case is for: local data image -----(mirror to)----> ceph image in order to configure the ceph backend, we have to use the ceph storage backend(only ceph), in this case, enable_ceph is 'yes' by default, so for all local images, we have to disable it with enable_ceph_xxx = no
> 
> hi @aliang123 , IMO, the purpose of this code update is that you'd like to add this case into your ceph test loop, right?

Yes, @zhencliu, when run tests with backend ceph, the system image should be on ceph by default. In our original case design, we just limit system storage to local storage, that's not reasonable, so correct it here.

Reply:
Thank you Zhenchao, Aihua for your explanation.
LGTM